Modules/IO/LSM/test/itkLSMImageIOTest.cxx

```diff
@@ -28,53 +28,50 @@ itkLSMImageIOTest(int argc, char * argv[])
 {
   if (argc < 3)
   {
-    std::cerr << "Usage: " << itkNameOfTestExecutableMacro(argv) << " LSM.lsm OutputImage.lsm\n";
+    std::cerr << "Missing Parameters." << std::endl;
+    std::cerr << "Usage: " << itkNameOfTestExecutableMacro(argv) << " inputFileName(*.lsm) ouputFileName(*.lsm)"
+              << std::endl;
     return EXIT_FAILURE;
   }
 
+  const unsigned int Dimensions = 2;
+
   using InputPixelType = itk::RGBPixel<unsigned char>;
-  using InputImageType = itk::Image<InputPixelType, 2>;
+  using InputImageType = itk::Image<InputPixelType, Dimensions>;
   using ReaderType = itk::ImageFileReader<InputImageType>;
   using ImageIOType = itk::LSMImageIO;
 
-  const char * filename = argv[1];
-  const char * outfilename = argv[2];
-
   ReaderType::Pointer reader = ReaderType::New();
-  reader->SetFileName(filename);
+  const char *        inputFileName = argv[1];
+  reader->SetFileName(inputFileName);
 
   ImageIOType::Pointer lsmImageIO = ImageIOType::New();
+
+  EXERCISE_BASIC_OBJECT_METHODS(lsmImageIO, LSMImageIO, TIFFImageIO);
+
+
   reader->SetImageIO(lsmImageIO);
 
-  try
-  {
-    reader->Update();
-  }
-  catch (const itk::ExceptionObject & e)
-  {
-    std::cerr << "exception in file reader " << std::endl;
-    std::cerr << e << std::endl;
-    return EXIT_FAILURE;
-  }
+  ITK_TRY_EXPECT_NO_EXCEPTION(reader->Update());
+
+
```

Comment:
Why so many empty lines around here?

Reply:
I grant that that is rather personal, but have always done it consistently that way; helps separating two main blocks to me in tests; type declarations prior to `ITK_EXERCISE_BASIC_OBJECT_METHODS`;  then `ITK_TRY_EXPECT_EXCEPTiON`/`ITK_TRY_EXPECT_NO_EXCEPTION`  blocks. For the rest of the cases, I only use a single empty line.